test/stream.ts

```diff
@@ -178,7 +182,7 @@ test('redirect response contains old url', withServer, async (t, server, got) =>
 	server.get('/redirect', redirectHandler);
 
 	const {requestUrl} = await pEvent(got.stream('redirect'), 'response');
-	t.is(requestUrl, `${server.url}/redirect`);
+	t.is(requestUrl as string, `${server.url}/redirect`);
```

Comment:
Is this necessary?

Reply:
Sorry, I forgot to remove it

Reply:
[`e7be039` (#1461)](https://github.com/sindresorhus/got/pull/1461/commits/e7be0398c6641f60e57f98f75b2812f97ad17611)